include/deal.II/base/quadrature_point_data.h

```diff
@@ -729,21 +743,27 @@ pack_cell_data(const CellIteratorType &                           cell,
     std::is_base_of<TransferableQuadraturePointData, DataType>::value,
     "User's DataType class should be derived from QPData");
 
+  if (!data_storage->stores_data(cell))
```

Comment:
In most places, we use this instead:
```suggestion
  if (data_storage->stores_data(cell) == false)
```